CHANGELOG.md

```diff
@@ -6,7 +6,15 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+
+## 5.8.0
+
 * `FEAT`: header displays element template icon found in XML ([#1012](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/1012))
+* `FEAT`: allow tooltip re-usablitity ([bpmn-io/properties-panel#321](https://github.com/bpmn-io/properties-panel/pull/321))

```

Comment:
How is this a feature of bpmn-js-properties-panel?

Reply:
Indeed it doesn't make sense here